content/reference/endpoints/bookings.md

```diff
@@ -59,41 +59,43 @@ payments          | Array    | Read      | Array of related Payments IDs |
-locked                  | String  | Write      | Booking's locked status. |
-notes                   | Text    | Read/Write      | Booking's notes. (1000 characters max) | maximum length: 6000
-reference               | String  | Read       | Booking's reference code. |
-rental_payback_to_owner | [Decimal](/reference/enums#formats) | Read/Write | Amount of money to pay back to owner (only rental, excluding fees). | Must be blank or greater than or equal to: 0 and lower than `initial_price`
-status                  | String  | Read       | Booking's status. |
-unavailable             | Boolean | Read/Write      | Booking's unavailable status, false by default. When set to true, marks the given period as unavailable. Commonly used for closed or maintenance periods. |
-external_reference      | String  | Read/Write | Booking's reference from the source channel. |
-cancelation_reason      | String  | Read/Write | Booking's cancelation reason (for writable part - it can be used in both update and cancelation endpoints) |
-cancelation_message_to_guest      | String  | Read/Write | Booking's cancelation message to guest (for writable part - only in cancelation endpoint) | 
-cancelation_message_to_channel    | String  | Read/Write | Booking's cancelation message to guest (for writable part - only in cancelation endpoint) |
-imported                | Boolean  | Read/Write | Whether this booking was imported from the external Channel or not | 
-reconciliation          | [Decimal](/reference/enums#formats  | Read/Write | Booking's reconciliation | 
-authorized_amount       | [Decimal](/reference/enums#formats  | Read | Booking's authorized amount | 
-paid_or_authorized_amount | [Decimal](/reference/enums#formats  | Read | Booking's paid or authorized amount |
+id                                             | Integer | Read       | Booking's id. |
+client_id                                      | Integer | Write      | Client id related to the Booking. |
+source_id                                      | Integer | Write      | Source id related to the Booking. |
+bookings_tag_ids                               | Array   | Write      | Array of ids of BookingsTags to be associated with Booking. |
+adults                                         | Integer | Read/Write      | Booking's number of adults. |
+booked                                         | Boolean | Read/Write      | Booking's booked status, false by default. When set to true, marks a regular booking. |
+channel_price                                  | Integer | Read/Write      | Price accepted by the traveler on the channel site, excluding traveler's fees (eg: Airbnb). |
+charge_damage_deposit_on_arrival               | Boolean    | Read/Write | Defines if charge damage deposit will be charged on arrival. | Can be `true` or `false` for Pro subscriptions, only `false` allowed when `damage_deposit_collection_method` is `"smily_damage_deposit_with_swikly"` |
```

Comment:
Let's move this below damage deposit? Also, we need to add that it can be only `true` for non-pro accounts when `with_smily`.

Reply:
Actually, it can only be `true` for PRO accounts when `with_smily`, isn't it like that?

Reply:
No, for light accounts.

Reply:
Ah, right!
https://github.com/BookingSync/BookingSync/blob/dbb6dc1d1023deb5ba2664eb42597fb8274bacf8/app/domain/reservations/damage_deposit/plan_config.rb#L24